examples/cloud-router-2-metal-nimf-redundant-connection/README.md

```diff
@@ -0,0 +1,13 @@
+# Fabric Cloud Router to Equinix Metal Redundant Connection
+
+This example shows how to leverage the [Fabric Cloud Router Connection Module](../../modules/cloud-router-connection/README.md)
+to create a Fabric Connection from a Fabric Cloud Router to Equinix Metal with two connections. A primary and secondary connection.
+
+It leverages the Equinix Terraform Provider, and the Fabric Cloud Router Connection
+Module to setup the connection based on the parameters you have provided to this example; or based on the pattern
+you see used in this example it will allow you to create a more specific use case for your own needs.
+
+See example usage below for details on how to use this example.
+
+<!-- Begin Example Usage (Do not edit contents) -->
```

Comment:
These don't belong. We're not leveraging them anymore. Please remove.

Reply:
Clarity on comment: we don't need these boundary declarations anymore `<!-- Begin Example Usage (Do not edit contents) -->`. They were for the previous shell script methods but now that we just use terraform-docs then we won't have any need for these. Hope that clears it up.

Reply:
Addressed with [commit](74747cd2cf07ab77431eb1de6c1f69e1749e578a)